Tests/Pcap++Test/main.cpp

```diff
@@ -209,6 +209,7 @@ int main(int argc, char* argv[])
 	PTF_RUN_TEST(TestPcapNgFileReadWriteAdv, "no_network;pcap;pcapng");
 	PTF_RUN_TEST(TestPcapFileReadLinkTypeIPv6, "no_network;pcap");
 	PTF_RUN_TEST(TestPcapFileReadLinkTypeIPv4, "no_network;pcap");
+	PTF_RUN_TEST(TestSolarisSnoopFileRead, "no_network;pcap");

```

Comment:
we can add a `snoop` tag: `"no_network;pcap;snoop"`

Reply:
Added in 27c7d04